test/test__helper.rb

```diff
@@ -33,12 +33,32 @@
   SimpleCov.formatter = SimpleCov::Formatter::Codecov
 end
 
+class Object
+  def truthy?
+    self && self != 'false'
+  end
+end
+
 module Minitest
+  require_relative '../lib/zold/log'
+
   class Test
     def test_log
-      require_relative '../lib/zold/log'
-      # @test_log = Zold::Log::Quiet.new
-      @test_log ||= Zold::Log::Verbose.new
+      @test_log ||= quiet_or_verbose_log
+    end
+
+    def quiet_or_verbose_log
```

Comment:
@mgrassotti `#verbosity_determined_by_env_log`? It doesn't help the reader to know that  it's either quiet or verbose.

Reply:
@ledestin ok cool. `#verbosity_by_env_log` ? just to keep it shorter?